traverson4j-hc5/src/test/java/uk/co/autotrader/traverson/http/entity/BodyPartEntityConverterTest.java

```diff
@@ -29,7 +29,7 @@ public void toEntity_GivenTextBody_ReturnsStringEntity() throws Exception {
 
         HttpEntity entity = converter.toEntity(new SimpleMultipartBody(new SimpleMultipartBody.BodyPart("file", "data".getBytes(StandardCharsets.UTF_8), "contentType", "filename")));
 
-        assertThat(entity.getContentType().getValue()).matches(Pattern.compile("multipart/form-data; boundary=.*"));
+        assertThat(entity.getContentType()).matches(Pattern.compile("multipart/form-data; charset=.* boundary=.*"));
```

Comment:
What charset will this default to? How come we wildcard the match?

Reply:
honestly dont remember even changing this! will sort